models/category.js

```diff
@@ -5,16 +5,17 @@ const categorySchema = new Schema(
   {
     name: {
       type: String,
-      unique: true
+      unique: true,
+      required: [true, 'Category name is required']
```

Comment:
Pay attention to the `consts/errors.js` file. There's already a `FIELD_CANNOT_BE_EMPTY` constant there with an appropriate message, so maybe it's better to use that? Or you can add your own message to that file. Same for the `color` field.

Reply:
Yep, thanks